alica_engine/include/engine/util/idFunctions.h

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
All other file names start with a capital letter so this one should too. E.g. IdFunctions.h